byron/ledger/formal-spec/properties.tex

```diff
@@ -210,5 +210,100 @@ \subsection{UTxO Properties}
   we have:
 
   $$ \var{reserves} + \balance{utxo} =  \var{reserves_0} + \balance{utxo_0} $$
+\end{property}
+
+\subsection{Update Properties}
+\label{sec:update-properties}
+
+\subsubsection{PVBUMP Properties}
+\label{sec:pvbump-properties}
+
+Property~\ref{PVBUMP-empty-future-adopt} states that when no proposals are
+available, the system remains in the same state it started with.
 
+\begin{property}[PVBUMP no change on empty future adoptions]\label{PVBUMP-empty-future-adopt}
```

Comment:
Nice. Here we need to universally quantify on `sn`, `pv`, and `pps`.

Also if you put the predicate that follows the `such that` part you can avoid awkward line breaks:

![image](https://user-images.githubusercontent.com/175315/58425827-25902280-809b-11e9-9326-c73a7ed8ba22.png)

Reply:
I'm not sure I'm following. Can you explain again?

Reply:
By quantifying I mean something like:

```
For all `sn`, `pv`, and `pps` such that

    ... blah ...

we have 

   ... the transition ...
```

Reply:
Done!